### PR TITLE
Audit color picker APIs and start docs

### PR DESCRIPTION
**Pull request by devongovett**

### CODE REVIEW

File: packages/@adobe/spectrum-css-temp/components/colorslider/skin.css [hunk 1]
```diff
@@ -3,35 +3,25 @@
...
-    .spectrum-ColorSlider-checkerboard {
-      &:before {
-        box-shadow: 0 0 0 var(--spectrum-colorslider-border-size) var(--spectrum-colorslider-border-color-disabled);
-      }
-    }
-
-    .spectrum-ColorSlider-gradient {
-      display: none
+    .spectrum-ColorSlider-track {
+      background: var(--spectrum-colorslider-fill-color-disabled) !important;
```
Comment (snowystinger) [inline packages/@adobe/spectrum-css-temp/components/colorslider/skin.css:23]: !?!? why do you hurt me like this?

File: packages/@adobe/spectrum-css-temp/components/colorwheel/skin.css [hunk 2]
```diff
@@ -5,13 +5,13 @@
 
 .spectrum-ColorWheel {
   &.is-disabled {
-    .spectrum-ColorWheel-outerCircle,
-    .spectrum-ColorWheel-innerCircle {
-      stroke: var(--spectrum-colorwheel-border-color-disabled);
-    }
+    .spectrum-ColorWheel-gradient {
+      background: var(--spectrum-colorwheel-fill-color-disabled) !important;
```
Comment (snowystinger) [inline packages/@adobe/spectrum-css-temp/components/colorwheel/skin.css:9]: same thing, do we just need higher specificity? or what's going on?

Comment (devongovett) [reply packages/@adobe/spectrum-css-temp/components/colorwheel/skin.css:9]: the gradient is on an inline style.

Comment (snowystinger) [reply packages/@adobe/spectrum-css-temp/components/colorwheel/skin.css:9]: hmmm could that inline style be a variable that we could apply here as multiple backgrounds?
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Using_multiple_backgrounds

or should disabled also be inline?

i'll keep thinking

Comment (devongovett) [reply packages/@adobe/spectrum-css-temp/components/colorwheel/skin.css:9]: yes, or we could not set it when disabled. however, this is all spectrum specific and the background is generated in aria...

**Review by dannify — APPROVED** [review]

**Pull request merged**